package ExceptionHandling;

public class excep1 {
    // public static void main(String args[]){
    //     int a=7;
    //     int b=0;
    //     try{
    //         System.out.println(a/b);
    //     }
    //     catch(Exception e){
    //         System.out.println(e.getMessage());


    //     }
    // }


    // public static void main(String args[]){
        
    //     try{
    //         int arr[]={1,2,3};
    //         System.out.println(arr[10]);
    //     }
    //     catch(Exception e){
    //         System.out.println(e.getMessage());


    //     }
    // }

    public static void main(String args[]){
        int a=7;
        int b=0;
        try{
            System.out.println(a/b);
        }
        catch(Exception e){
            System.out.println(e.getMessage());


        }
    }
    
}
